src/NLog/Internal/StreamHelpers.cs

```diff
@@ -96,7 +95,7 @@ public static void CopyWithOffset(this Stream input, Stream output, int offset)
         {
             if (offset < 0)
             {
-                throw new ArgumentException("negative offset");
+                throw new ArgumentException("cannot be negative", nameof(offset));
```

Comment:
Maybe this should be a ArgumentOutOfRangeException?

Reply:
Just working on removing false positives. So real errors are not hidding in the noise.